docs/variants/novacustom_ns5x_7x_12th_gen/building-manual.md

```diff
@@ -0,0 +1,57 @@
+# Building manual
+
+## Intro
+
+This document describes the procedure for compiling coreboot for NovaCustom
+NS5xPU/NS7xPU.
+
+## Requirements
+
+- Docker
+    + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+- Git
+
+## Procedure
+
+The easiest way to build coreboot is to use the official Docker image.
+
+1. Obtain the image:
```

Comment:
These instructions are out-of-date as of TGL release v1.3.0, please see: https://github.com/Dasharo/docs/pull/311

Reply:
Fixed